addons/l10n_tr_nilvera_einvoice/models/account_edi_xml_ubl_tr.py

```diff
@@ -3,6 +3,7 @@
 from num2words import num2words
 
 from odoo import api, models
+from odoo.tools import html2plaintext
```

Comment:
Please follow imports orders according to [guidelines](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports).